pandas/core/indexes/base.py

```diff
@@ -3767,8 +3767,8 @@ def drop(self, labels, errors='raise'):
         mask = indexer == -1
         if mask.any():
             if errors != 'ignore':
-                raise ValueError('labels %s not contained in axis' %
-                                 labels[mask])
```

Comment:
need to update the doc-strings for all (Index,Series,DataFrame,Panel) for drop to change ValueError -> KeyError in the Raises section (or add it if its not there)